### PR TITLE
MF-44

### PR DESCRIPTION
**Pull request by Andu033**

### CODE REVIEW

File: src/button-icons/buttons-icons.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.omrs-btn-icon-medium {
+  width: 24px;
```
Comment (fatmali) [inline src/button-icons/buttons-icons.css]: @Andu033 we are using `rem` instead of `px`. `1rem` = `16px`

File: src/styleguide-entry.js [hunk 2]
```diff
@@ -9,3 +9,4 @@ import "./logo/logo.js";
 import "./icons/icons.js";
 import "./inputs/input.css";
 import "./main-content/main-content.css";
+import "./button-icons/buttons-icons.css";
```
Comment (fatmali) [inline src/styleguide-entry.js]: nit: rename to `button-icon.css`

File: src/button-icons/buttons-icons.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  height: 24px;
+  text-align: center !important;
+  background: transparent;
+  border: none !important;
```
Comment (fatmali) [inline src/button-icons/buttons-icons.css]: using `!important` might not allow overriding this property on the client. @joeldenning what are your thoughts on this?

File: src/button-icons/buttons-icons.stories.html [hunk 4]
```diff
@@ -0,0 +1,83 @@
+<article>
+  <h2>Button icons</h2>
+  <div>
+    <table>
+      <tr>
+        <div>
```
Comment (joeldenning) [inline src/button-icons/buttons-icons.stories.html]: nit: this div in between the `<tr>` and the `<td>` is sorta weird. There shouldn't be a need for it. Does something break if it's removed?

File: src/button-icons/buttons-icons.stories.html [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+                <use xlink:href="#omrs-icon-search"></use>
+              </svg>
+            </button>
+          </td>
+        </div>
+      </tr>
+    </table>
+  </div>
+
+  <div class="stories-example lang-html">
```
Comment (joeldenning) [inline src/button-icons/buttons-icons.stories.html]: Would be cool to add knobs that let the user adjust the icon and size with select elements. See the following links for how this could be done. I'd recommend exporting the list of icons from the icons.stories.js file so that it could be reused here.

- https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/icons/icons.stories.js#L23
- https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/icons/icons.stories.html#L24

File: src/button-icons/button-icons.css [hunk 6]
```diff
@@ -0,0 +1,64 @@
+.omrs-btn-icon-medium {
+  width: 2rem;
+  height: 2rem;
+  text-align: center;
+  background: transparent;
+  border: none;
+  font-size: 0;
```
Comment (joeldenning) [inline src/button-icons/button-icons.css]: Why is this here? There's no text inside of the buttons, right, so why change the font size?

File: src/button-icons/button-icons.css [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  text-align: center;
```
Comment (joeldenning) [inline src/button-icons/button-icons.css]: This is a duplicate of line 4.

```suggestion
```

File: src/button-icons/buttons-icons.stories.html (hunk 4 above)
Comment (Andu033) [reply src/button-icons/buttons-icons.stories.html]: I removed it and it works fine.

File: src/button-icons/button-icons.css (hunk 6 above)
Comment (Andu033) [reply src/button-icons/button-icons.css]: Removed it. I forgot to clean a little.

File: src/button-icons/button-icons.css (hunk 7 above)
Comment (Andu033) [reply src/button-icons/button-icons.css]: I removed it.

File: src/button-icons/buttons-icons.stories.html (hunk 5 above)
Comment (Andu033) [reply src/button-icons/buttons-icons.stories.html]: I added. Thank you for links.

Comment (Andu033) [reply src/button-icons/buttons-icons.stories.html]: I copied the list of icons because i don't have a pretty way to export that list. Can you suggest one?

Comment (joeldenning) [reply src/button-icons/buttons-icons.stories.html]: Yeah see the following links for how to import/export things from one file to another:

- https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/colors/colors.stories.js#L8
- https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/icons/icons.stories.js#L7
